test/blackbox-tests/test-cases/cram/shell-opt-dune-lang-version.t

```diff
@@ -0,0 +1,42 @@
+Check that (shell ..) option on (cram ..) stanza fail before (lang dune 3.10)
+
+  $ echo '(lang dune 3.9)' > dune-project
+
+  $ cat > dune <<EOF
+  > (cram (shell :system))
+  > EOF
+
+  $ cat > foo.t <<EOF
+  >   $ echo "foo from foo.t"
+  > EOF
+
+  $ dune runtest -f
+  File "dune", line 1, characters 6-21:
+  1 | (cram (shell :system))
+            ^^^^^^^^^^^^^^^
+  Error: 'shell' is only available since version 3.10 of the dune language.
+  Please update your dune-project file to have (lang dune 3.10).
+  [1]
+
+Check that (shell ..) option on (cram ..) user action fail before (lang dune 3.10)
+
+  $ echo '(lang dune 3.9)' > dune-project
+
+  $ cat > dune <<EOF
+  > (rule
+  >   (target foo.cram.corrected)
+  >   (action (cram foo.cram (shell :bash)))
+  > )
+  > EOF
+
+  $ cat > foo.cram <<EOF
+  >   $ echo "foo from foo.cram"
+  > EOF
+
+  $ dune runtest -f
+  File "dune", line 3, characters 25-38:
+  3 |   (action (cram foo.cram (shell :bash)))
+                               ^^^^^^^^^^^^^
+  Error: 'shell' is only available since version 3.10 of the dune language.
+  Please update your dune-project file to have (lang dune 3.10).
```

Comment:
```suggestion
  Error: 'shell' is only available since version 3.10 of the Dune language.
  Please update your `dune-project` file to have `(lang dune 3.10)`.
```

Reply:
Not sure if these can be changed in the doc, as I know sometimes its the actual output formatted that way. If this is the case, please feel free to ignore or dismiss.

Reply:
Yes, I believe this is the output of running the test so changing it here will result in this test case to fail.

If a fix is desired, I think an issue should be open and it should be addressed by another PR (as fixing this would be outside of the scope of this PR)